web/src/client/storage.js

```diff
@@ -25,6 +25,7 @@ import { applyMixin, withDBus } from "./mixins";
 const STORAGE_PROPOSAL_IFACE = "org.opensuse.DInstaller.Storage.Proposal1";
 const STORAGE_ACTIONS_IFACE = "org.opensuse.DInstaller.Storage.Actions1";
 const ACTIONS_PATH = "/org/opensuse/DInstaller/Storage/Actions1";
+const TARGET_PATH = "/org/opensuse/DInstaller/Storage/Proposal1";
```

Comment:
NP:

```suggestion
const STORAGE_PROPOSAL_PATH = "/org/opensuse/DInstaller/Storage/Proposal1";
```

Reply:
Or

```suggestion
const PROPOSAL_PATH = "/org/opensuse/DInstaller/Storage/Proposal1";
```

I'd go for that and even change the STORAGE_XYZ_IFACE ones for not using the STORAGE prefix.